Libplanet/Blocks/BlockHeader.cs

```diff
@@ -59,6 +64,10 @@ namespace Libplanet.Blocks
         /// Goes to the <see cref="TxHash"/>.</param>
         /// <param name="hash">The hash of the <see cref="Block{T}"/>.
         /// Goes to the <see cref="Hash"/>.</param>
+        /// <param name="preEvaluationHash">The hash of pre-commit block.
+        /// using for check validate.</param>
+        /// <param name="evaluationDigest">The hash of the <see cref="ActionEvaluation"/>s.
+        /// <seealso cref="EvaluationDigest"/>.</param>
```

Comment:
Follow the consistent docs to the `Block<T>()` constructor's parameter.

`<seealso>` cannot be inside `<param>` (or any other elements like `<summary>`).

```suggestion
        /// <param name="evaluationDigest">The hash derived from the result states of every
        /// <see cref="Transaction{T}.Actions"/> of the block's entire transactions.</param>
```